parquet/src/file/statistics.rs

```diff
@@ -416,9 +429,20 @@ impl Statistics {
     /// Returns number of null values for the column, if known.
     /// Note that this includes all nulls when column is part of the complex type.
     ///
-    /// Note this API returns Some(0) even if the null count was not present
-    /// in the statistics.
-    /// See <https://github.com/apache/arrow-rs/pull/6216/files>
+    /// Note: Versions of this library prior to `53.0.0` returned 0 if the null count was
+    /// not available. This method returns `None` in that case.
+    ///
+    /// Also, versions of this library prior to `53.0.0` did not store the null count in the
```

Comment:
```suggestion
    /// Also, versions of this library prior to `53.1.0` did not store the null count in the
```
The write changes just missed the 53.0.0 cutoff it seems.